cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -29,10 +29,15 @@ desiredState:
                     "-advertise-client-urls=http://127.0.0.1:4001",
             ]
           - name: kube2sky
-            image: gcr.io/google_containers/kube2sky:1.1
+            image: gcr.io/google-containers/kube2sky:1.2
```

Comment:
@thockin it needs to be fix and is pn my list for today
On Wed, Apr 22, 2015 at 2:11 AM CJ Cullen <notifications@github.com> wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7154#discussion_r28846947>
> :
>
> > @@ -29,10 +29,15 @@ desiredState:
> >                      "-advertise-client-urls=http://127.0.0.1:4001",
> >              ]
> >            - name: kube2sky
> > -            image: gcr.io/google_containers/kube2sky:1.1
> > +            image: gcr.io/google-containers/kube2sky:1.2
>
> Yes, already pushed (hopefully nobody is using latest?).
> And fixed the _ vs. -
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7154/files#r28846947>
> .
>